src/modules/undo-manager.coffee

```diff
@@ -33,7 +33,10 @@ class UndoManager
     )
     @quill.on(@quill.constructor.events.TEXT_CHANGE, (delta, origin) =>
```

Comment:
Can you rename `origin` to `source`? That's the more common name in other parts of the codebase.